### PR TITLE
[ES] Add `Approach_rate`

### PR DESCRIPTION
**Pull request by Ekkusu**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Review by agatemosu — APPROVED** [review]

File: wiki/Beatmap/Approach_rate/es.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+La cantidad de tiempo que tarda para que el objeto de golpeo desaparezca por completo, también dependerá de la velocidad de aproximación:
+
+- AR < 5: `desvanecer = 800ms + 400ms * (5 - AR) / 5`
+- AR = 5: `desvanecer = 800ms`
+- AR > 5: `desvanecer = 800ms - 500ms * (AR - 5) / 5`
+
+## Efectos de Mod
+
+Estos son los cuatros mods pueden alterar la tasa de aproximación cuando se activan:
```
Comment (agatemosu) [inline wiki/Beatmap/Approach_rate/es.md:45]:
```suggestion
Estos son los cuatro mods pueden alterar la tasa de aproximación cuando se activan:
```

File: wiki/Beatmap/Approach_rate/es.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- [Easy](/wiki/Game_modifier/Easy): Reduce a la mitad el valor AR.
+- [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multiplica el valor AR por 1.4, hasta un máximo de 10.
+- [Double Time](/wiki/Game_modifier/Double_Time): El valor AR no se ve afectado, pero debido al aumento del 50% en la velocidad del juego, los objetos de golpeo permanecen un 33% menos de tiempo.
+- [Half Time](/wiki/Game_modifier/Half_Time): El valor AR no se ve afectado, pero debido a la disminución del 25% en la velocidad del juego, los objetos de golpeo permanecen un 33% más de tiempo.
+
+Si bien el Half Time y Double Time no cambian el valor del AR, la diferencia de velocidad conduce a un cambio aparente del AR. Los AR del HT/DT se denominan comúnmente en términos de su valor percibido. Por ejemplo, "AR8+DT" puede también puede escribirse como "AR9.6".
+
+## osu!taiko y osu!mania
+
+La tasa de aproximación es visible cuando se visualiza la información de un mapa de [osu!taiko](/wiki/Game_mode/osu!taiko) o [osu!mania](/wiki/Game_mode/osu!mania), pero esta no afectara al juego. La velocidad de desplazamiento de estos modos esta determinado por otros factores.
```
Comment (agatemosu) [inline wiki/Beatmap/Approach_rate/es.md:56]:
```suggestion
La tasa de aproximación es visible cuando se visualiza la información de un mapa de [osu!taiko](/wiki/Game_mode/osu!taiko) o [osu!mania](/wiki/Game_mode/osu!mania), pero esta no afectara al juego. La velocidad de desplazamiento de estos modos está determinado por otros factores.
```

File: wiki/Beatmap/Approach_rate/es.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+## Efectos de Mod
+
+Estos son los cuatros mods pueden alterar la tasa de aproximación cuando se activan:
+
+- [Easy](/wiki/Game_modifier/Easy): Reduce a la mitad el valor AR.
+- [Hard Rock](/wiki/Game_modifier/Hard_Rock): Multiplica el valor AR por 1.4, hasta un máximo de 10.
+- [Double Time](/wiki/Game_modifier/Double_Time): El valor AR no se ve afectado, pero debido al aumento del 50% en la velocidad del juego, los objetos de golpeo permanecen un 33% menos de tiempo.
+- [Half Time](/wiki/Game_modifier/Half_Time): El valor AR no se ve afectado, pero debido a la disminución del 25% en la velocidad del juego, los objetos de golpeo permanecen un 33% más de tiempo.
+
+Si bien el Half Time y Double Time no cambian el valor del AR, la diferencia de velocidad conduce a un cambio aparente del AR. Los AR del HT/DT se denominan comúnmente en términos de su valor percibido. Por ejemplo, "AR8+DT" puede también puede escribirse como "AR9.6".
```
Comment (agatemosu) [inline wiki/Beatmap/Approach_rate/es.md:52]:
```suggestion
Si bien Half Time y Double Time no cambian el valor del AR, la diferencia de velocidad conduce a un cambio aparente del AR. Los AR del HT/DT se denominan comúnmente en términos de su valor percibido. Por ejemplo, "AR8+DT" puede también puede escribirse como "AR9.6".
```

Comment (Ekkusu) [reply wiki/Beatmap/Approach_rate/es.md:52]: Los dos cambios anteriores fueron acertados, pero este es el único que no me convence, lo dejare así mientras tanto y luego tomo una decisión.

**Comment by SrIzan10** [comment]
update?

**Comment by agatemosu** [comment]
[**Use short and meaningful commit messages**, as they let others know what's in the box. Something like `Rewrite the section about jump patterns` says a lot more than `Update en.md`.](https://osu.ppy.sh/wiki/en/osu%21_wiki/Contribution_guide/Best_practices#making-edits)

**Comment by SrIzan10** [comment]
naa, me refiero si a ver alguien aprueba esto

**Review by SrIzan10 — APPROVED** [review]

**Comment by agatemosu** [comment]
> naa, me refiero si a ver alguien aprueba esto

At the end of the document it says TODO, that's why I didn't approve it

File: wiki/Beatmap/Approach_rate/es.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+  - lectura
+---
+
+# Tasa de aproximación
+
+*Para conocer los valores sugeridos del AR, consulte: [Criterios de ranking](/wiki/Ranking_Criteria)*
+
+La **Tasa de aproximación** (***AR***) es una configuración de la dificultad de un [mapa](/wiki/Beatmap) que define cuando los [objetos de golpeo](/wiki/Hit_object) empiezan a desvanecerse en relación con cuando deben que ser golpeados o recolectados. Existe solo en [osu!](/wiki/Game_mode/osu!) y [osu!catch](/wiki/Game_mode/osu!catch).
+
+Los valores del AR varían de 0 a 10. Las tasas de aproximación altas significan que los objetos de golpeo se mostraran en un periodo más corto de tiempo, lo que dará menos tiempo al jugador para reaccionar. Por otro lado, las tasas de aproximación bajas significan más tiempo de reacción, lo que puede resultar en una cantidad abrumadora de objetos de golpeo que aparecerán en la pantalla a la vez.
```
[the remark this answers is not in the document]
Comment (agatemosu) [reply wiki/Beatmap/Approach_rate/es.md]: Is 10

File: wiki/Beatmap/Approach_rate/es.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+
+Si bien Half Time y Double Time no cambian el valor del AR, la diferencia de velocidad conduce a un cambio aparente del AR. Los AR del HT/DT se denominan comúnmente en términos de su valor percibido. Por ejemplo, "AR8+DT" puede también puede escribirse como "AR9.6".
+
+## osu!taiko y osu!mania
+
+La velocidad de aproximación es visible cuando se visualiza la información de un mapa de [osu!taiko](/wiki/Game_mode/osu!taiko) o [osu!mania](/wiki/Game_mode/osu!mania), pero esta no afectara al juego. La velocidad de desplazamiento de estos modos está determinado por otros factores.
+
+<!-- TODO: Insert lots of links -->
+
+<!-- TODO: a chart would be nice for comparing speeds/values with mods -->
```
Comment (agatemosu) [inline wiki/Beatmap/Approach_rate/es.md:60]:
```suggestion
```

Comment (Ekkusu) [reply wiki/Beatmap/Approach_rate/es.md:60]: I don't know what that is for, but it was in the code of the article in English, so I copied it (https://github.com/ppy/osu-wiki/blob/master/wiki/Beatmap/Approach_rate/en.md?plain=1#:~:text=%3C!%2D%2D%20TODO%3A%20Insert,with%20mods%20%2D%2D%3E)

Anyway, I'll ask if it's necessary

Comment (agatemosu) [reply wiki/Beatmap/Approach_rate/es.md:60]: Ah ok, I did not see it

**Review by agatemosu — APPROVED** [review]

File: wiki/Beatmap/Approach_rate/es.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (Ekkusu) [reply wiki/Beatmap/Approach_rate/es.md]: Yes, it is up to 10 in the English article. https://github.com/ppy/osu-wiki/blob/master/wiki/Beatmap/Approach_rate/en.md?plain=1#:~:text=AR%20values%20range%20from%200%20to%2010

**Review by TPGPL — APPROVED** [review]

**Pull request merged**